### DISCUSSION TITLE
What is the status of other fiber tracing features like tree rendering?

### DISCUSSION THREAD

**Discussion by lilac** [typelevel/cats-effect#1516] [Q&A] [answered]
The tracing doc page mentions that "fiber tracing" has a lot of features but only bolded names have been merged to master. But the status of others are not mentioned.

Have they been developed but not yet merged, or are just in plan?

**Comment by djspiewak** [comment, marked as answer]
Replied [in Gitter](https://gitter.im/typelevel/cats-effect?at=5fdcca4d2084ee4b785b3cc2) before I saw this, but for posterity, the short answer is: they're coming! Cats Effect 3 work took priority throughout most of this year, and that also has the pleasing side-effect of making some of these features easier to implement due to the nature of fibers in CE3.

**Reply by lilac** [reply]
Thanks
